src/annotator/sidebar.js

```diff
@@ -449,4 +450,22 @@ export default class Sidebar extends Guest {
   setAllVisibleHighlights(shouldShowHighlights) {
     this.crossframe.call('setVisibleHighlights', shouldShowHighlights);
   }
+
+  /**
+   * Shows the sidebar's controls
+   */
+  showControls() {
```

Comment:
This and `hideControls` are well-named 👍🏻 

I'm having trouble drawing a table of possible states in my head. Does anything prevent controls from being hidden when the sidebar is open?

Reply:
Yesterday, I realised there is an alternative way to hide the controls in the sidebar that is used when the `externalContainer` option is used. It maybe better to investigate to reuse that options so we don't end up with two alternative ways to achieve the same thing.

Reply:
I tried to reuse the previous mechanism, using `this.toolbarWidth`, but it serves a different purpose and it is not reusable.

Reply:
>Does anything prevent controls from being hidden when the sidebar is open?

I can't think on a state where the sidebar is open but the controls are hidden. The logic to show/hide controls and open/close the sidebar are only in two functions and triggered by two events `open/closeNotebook`.

On  `openNotebook` the sidebars (1) hides its controls and (2) closes the bar. These two actions always should happen together.

On `closeNotebook` event, the sidebar (1) shows its controls and (2) opens the bar. (1) must always happen. (2) maybe optional (in the future we may want to restore the UI state [previous to the opening of the notebook] if we device a different way to open the notebook that doesn't require the sidebar to be opened).